cell.go

```diff
@@ -143,6 +147,33 @@ func (cb *CellBuffer) SetDirty(x, y int, dirty bool) {
 	}
 }
 
+// LockCell adds a lock to the cell, effectively marking it as clean until all
+// locks on it are cleared
```

Comment:
I really feel like this comment could do with a bit more explanatory text.  The idea of "locking" the cell, and what this means, is a little strange, and if it's not obvious to me I presume it won't be obvious to other users of this API.

Reply:
How about something like this:

```go
// LockCell locks a cell from being drawn, effectively marking it "clean" until
// the lock is removed. This can be used to prevent tcell from drawing a given
// cell, even if the underlying content has changed. For example, when drawing a
// sixel graphic directly to a TTY screen an implementer must lock the region
// underneath the graphic to prevent tcell from drawing on top of the graphic.
```

Reply:
@gdamore Friendly bump :)

Reply:
Can you go ahead and make that change?